app/full_text_search.py

```diff
@@ -160,17 +160,24 @@ def create_fields_for_rank(field_name, value):
     return fields
 
 
-def create_full_name_without_space(given_name, family_name):
+def create_full_name_without_space(given_names, family_names):
     """
     Creates full name without white space.
     Returns:
-        if given_name and family_name: ('given_name + family_name',
-                                        'family_name + given_name')
+        if given_name and family_name: [('given_name + family_name',
```

Comment:
This looks like the function returns a single element. Please fix. You can explain in English instead of pseudo code.